src/NumberIncrement/index.js

```diff
@@ -0,0 +1,174 @@
+import React, { Component } from "react";
+import withStyles from "react-jss";
+import classNames from "classnames";
+import Remove from "elevate-ui-icons/Remove";
+import Add from "elevate-ui-icons/Add";
+
+import Scaffold from "../Scaffold";
+
+type Props = {
+  classes: Object,
+  /**
+   * Accepts any classNames to be passed down to the component.
+   */
+  className: string,
+  field: Object,
+  form: Object,
+  id: string,
+  /**
+   * Text input to be used as the label for inside the component.
+   */
+  label: string,
+  theme: Object,
+  /**
+   * Use the scaffold for the input or if the input should just be the raw input.
+   */
+  withScaffold: boolean,
+};
+
+/**
+ * Number input that can be adjusted incrementally
+ */
+class NumberIncrement extends Component<Props> {
+  handleValueChange = (direction) => {
+    direction === "increase" ? this.input.stepUp() : this.input.stepDown();
```

Comment:
Super nitpicky– can we change where we are passing `increase/decrease` to be `increment/decrement`?

I know a lot of tutorials that are teaching the counter concept use increment/decrement, plus it'll make it match the name of the component.